packs/data/classfeatures.db/antipaladin.json

```diff
@@ -10,7 +10,7 @@
+            "value": "<p>You're dishonorable, dishonest, and committed to breaking the false hopes of kindness. You gain the @UUID[Compendium.pf2e.actionspf2e.Destructive Vengeance]{Destructive Vengeance} champion's reaction and the <em>@UUID[Compendium.pf2e.spells-srd.Touch of Corruption]{Touch of Corruption}</em> devotion spell. After the tenets of evil, add these tenets.</p>\n<ul>\n<li>You must not bind yourself with any law or oath beyond that of your code (though you can still select a champion feat with the oath trait). Act dishonorably, take advantage of others, lie, cheat, and steal to get what you want.</li>\n<li>You must destroy that which offends you and that which stands in your way, including-and perhaps especially-the forces of good and law that oppose you. This tenet doesn't force you to take action immediately if it could mean your destruction, nor does it require you to destroy something that might be useful if corrupted toward your ends.</li>\n</ul>\n<h2 class=\"title\">Champion's Reaction</h2>\n<h3 class=\"title\"><span style=\"font-size:14px;font-weight:500\">@UUID[Compendium.pf2e.actionspf2e.Destructive Vengeance]{Destructive Vengeance} <span class=\"pf2-icon\">r</span></span></h3>\n<p><strong>Trigger</strong> An enemy within 15 feet damages you</p>\n<hr />\n<p>Bloodshed begets bloodshed as you drag your enemy toward oblivion. You increase the amount of damage you take by 1d6, and you deal 1d6 damage to the triggering enemy, choosing evil or negative damage to deal to the enemy each time you use this reaction. In addition, until the end of your next turn, your @UUID[Compendium.pf2e.actionspf2e.Strike]{Strikes} against the triggering creature deal 2 extra damage of the type you chose.</p>\n<p>The damage you take and deal when you use this reaction increases to 2d6 at 5th level, 3d6 at 9th level, 4d6 at 12th level, 5d6 at 16th level, and 6d6 at 19th level. The extra damage on your Strikes increases to 4 at 9th level and 6 at 16th level.</p>\n<h2 class=\"title\">Divine Smite<span style=\"float:right\">Level 9</span></h2>\n<p>An enemy damaged by the initial damage from your Destructive Vengeance also takes [[/r @actor.abilities.cha.mod[persistent,evil]]]{Persistent Evil Damage} equal to your Charisma modifier. This applies only to the damage the reaction itself deals to the triggering creature, not the damage you deal with subsequent @UUID[Compendium.pf2e.actionspf2e.Strike]{Strikes}.</p>\n<h2 class=\"title\">Exalt<span style=\"float:right\">Level 11</span></h2>\n<p>When you use Destructive Vengeance, each enemy within 15 feet of you other than the triggering creature takes half the damage you deal to the triggering enemy, of the same damage type you chose.</p>"
```

Comment:
```suggestion
            "value": "<p>You're dishonorable, dishonest, and committed to breaking the false hopes of kindness. You gain the @UUID[Compendium.pf2e.actionspf2e.Destructive Vengeance]{Destructive Vengeance} champion's reaction and the <em>@UUID[Compendium.pf2e.spells-srd.Touch of Corruption]{Touch of Corruption}</em> devotion spell. After the tenets of evil, add these tenets.</p>\n<ul>\n<li>You must not bind yourself with any law or oath beyond that of your code (though you can still select a champion feat with the oath trait). Act dishonorably, take advantage of others, lie, cheat, and steal to get what you want.</li>\n<li>You must destroy that which offends you and that which stands in your way, including-and perhaps especially-the forces of good and law that oppose you. This tenet doesn't force you to take action immediately if it could mean your destruction, nor does it require you to destroy something that might be useful if corrupted toward your ends.</li>\n</ul>\n<h2 class=\"title\">Champion's Reaction</h2>\n<h3 class=\"title\">@UUID[Compendium.pf2e.actionspf2e.Destructive Vengeance]{Destructive Vengeance} <span class=\"pf2-icon\">r</span></h3>\n<p><strong>Trigger</strong> An enemy within 15 feet damages you</p>\n<hr />\n<p>Bloodshed begets bloodshed as you drag your enemy toward oblivion. You increase the amount of damage you take by 1d6, and you deal 1d6 damage to the triggering enemy, choosing evil or negative damage to deal to the enemy each time you use this reaction. In addition, until the end of your next turn, your @UUID[Compendium.pf2e.actionspf2e.Strike]{Strikes} against the triggering creature deal 2 extra damage of the type you chose.</p>\n<p>The damage you take and deal when you use this reaction increases to 2d6 at 5th level, 3d6 at 9th level, 4d6 at 12th level, 5d6 at 16th level, and 6d6 at 19th level. The extra damage on your Strikes increases to 4 at 9th level and 6 at 16th level.</p>\n<h2 class=\"title\">Divine Smite<span style=\"float:right\">Level 9</span></h2>\n<p>An enemy damaged by the initial damage from your Destructive Vengeance also takes [[/r @actor.abilities.cha.mod[persistent,evil]]]{Persistent Evil Damage} equal to your Charisma modifier. This applies only to the damage the reaction itself deals to the triggering creature, not the damage you deal with subsequent @UUID[Compendium.pf2e.actionspf2e.Strike]{Strikes}.</p>\n<h2 class=\"title\">Exalt<span style=\"float:right\">Level 11</span></h2>\n<p>When you use Destructive Vengeance, each enemy within 15 feet of you other than the triggering creature takes half the damage you deal to the triggering enemy, of the same damage type you chose.</p>"
```